### ISSUE TITLE
Can I use LightGBM with pre-trained model like Keras?

### ISSUE THREAD

**Issue by inoue0426** [Microsoft/LightGBM#1929]
I want to use LightGBM with the pre-trained model like fine-tuning, interactive change for new dataset and the number of target label.
I thought the refit method is able to use like fine-tuning.

Here, I tried to use the refit method with the pre-trained model.
model = lgb_model.booster_.refit(X_train, y_train)

lgb_model is "LGBMClassifier(boosting_type='gbdt', class_weight=None, colsample_bytree=1.0,
        device='gpu', importance_type='split', learning_rate=0.1,
        max_bin=255, max_depth=7, min_child_samples=20,
        min_child_weight=0.001, min_data_in_leaf=15, min_split_gain=0.0,
        n_estimators=100, n_jobs=-1, num_iterations=10000, num_leaves=90,
        objective='multiclass', random_state=None, reg_alpha=0.0,
        reg_lambda=0.0, silent=True, subsample=1.0,
        subsample_for_bin=200000, subsample_freq=0)"

The pre-trained model's classes_ is 34 and new y_train's unique number is 20.
Therefore, I wanted the new model's class to change the classes from 34 to 20.

Previous issue, #1473 , said 'It is just faster than retrain'.
So I can't use LightGBM with the pre-trained model like fine-tuning, right?

**Comment by guolinke** [comment]
yes, I think the fine-tuned does not suit your case, for the number of classes changes.

**Comment by inoue0426** [comment]
I see.
Thanks!
